docs/ref/django-admin.txt

```diff
@@ -395,28 +409,30 @@ table's lifecycle, you'll need to change the
 :attr:`~django.db.models.Options.managed` option to ``True`` (or simply remove
 it because ``True`` is its default value).
 
-The :djadminopt:`--database` option may be used to specify the
-database to introspect.
+.. django-admin-option:: --database <db_alias>
+
+Specifies the database to introspect. Defaults to ``default``.
 
-loaddata <fixture fixture ...>
+loaddata [fixture fixture ...]
```

Comment:
Brackets are for optional args. I'm roughly copying these from `--help` so this should be `fixture [fixture ...]`. Maybe we should omit the angle brackets as that command does and use things like `--database DATABASE` so there's less translation/thinking involved when writing these up.